### PR TITLE
[GES] Fix typos

### PR DESCRIPTION
**Pull request by Pesteves2002**

### CODE REVIEW

File: content/ges/0002-organizacoes-recursos-humanos.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ Presidente da República > Primeiro Ministro > Ministros > Secretários > Tu.
 
    - **Vantagens**: Especialização dos empregados. Economias de Escala.
    - **Desvantagens**: Fraca comunicação dos departamentos.
-   - **Características**: Estabilidade, Eficiência e Empresas de grande Dimensão.
+   - **Características**: Estabilidade, Eficiência em Empresas de grande Dimensão.
```
Comment (diogotcorreia) [inline content/ges/0002-organizacoes-recursos-humanos.md:99]: I'm not sure if this is correct, from my POV this means "normally used by big companies", which was what the previous content meant.
@randomicecube what do you think?

File: content/ges/0004-informacao-financeira.md [hunk 2]
```diff
@@ -363,26 +366,26 @@ $$
...
 
-![Empresa Comercial](./assets/0004-empresa-comercial.png)
+![Empresa Comercial](./assets/0004-empresa-comercial.png#dark=1)
 
 ### Empresa Industrial - Custeio Total
 
 - Mais difícil de ver o custo do produto.
 
-- **Custeio Total**: A empresa valoriza a produção como o total dos CIF (custos indiretos de fabrico).
+- **Custeio Total**: A empresa valoriza a produção como o total dos CIF (Custos Fndiretos de Fabrico).
```
Comment (diogotcorreia) [inline content/ges/0004-informacao-financeira.md:378]:
```suggestion
- **Custeio Total**: A empresa valoriza a produção como o total dos CIF (Custos Indiretos de Fabrico).
```

**Review by diogotcorreia — CHANGES_REQUESTED** [review]

File: content/ges/0002-organizacoes-recursos-humanos.md (hunk 1 above)
Comment (randomicecube) [reply content/ges/0002-organizacoes-recursos-humanos.md:99]: Agreed, I think the previous content was correct.

**Review by Pesteves2002 — COMMENTED** [review]
Good

**Review by diogotcorreia — APPROVED** [review]

**Pull request merged**